### ISSUE TITLE
Show indentation

### ISSUE THREAD

**Issue by wmertens** [richie5um/QLZip#1]
stripping the paths and instead using indentation would make the preview easier to read and also less bytes to show for big zipfiles

**Comment by richie5um** [comment]
Good idea. It’s been a while since I did any work on this plugin. I’ll try and take a look.

**Comment by richie5um** [comment]
I'm wondering if I can actually make it a proper treeview. WIP test project over here: https://github.com/richie5um/ZipToHTML
